db/migrate/20231201015717_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[7.1]
```

Comment:
- [ ] [REQUIRED] Due to the rails naming convention as I have recommended in the model, the right name for the migration create table is `greetings` not `messages`. Let's resolve this.